lib/tmuxinator/assets/sample.yml

```diff
@@ -19,7 +19,10 @@ root: ~/
 # tmux_command: byobu
 
 # Specifies (by name or index) which window will be selected on project startup. If not set, the first window is used.
-# startup_window: logs
```

Comment:
I modified this option, so that the startup_pane options below can actually select one out of two window. 👍 ❓